docs/build_stage_compile_settings.md

```diff
@@ -0,0 +1,33 @@
+# Compile Settings for Build Stages
+This doc was last updated: 11/3/20
+
+This doc describes the different compiler options for the Jenkins builds. If a cell is left blank it means that the option is not specified thus uses the default.
+
+## Terrier build
```

Comment:
Nit: terrier -> NoisePage. Though the URL has to stay the same, yeah.

Reply:
I think we should probably call it terrier build until we actually change the name in Jenkins, otherwise, I think it would be confusing.